app/konnect/network-resiliency.md

```diff
@@ -36,6 +36,11 @@ plane. The node sends a heartbeat to the control plane every 30 seconds to
 keep the connection alive. If it receives no answer, it tries to reconnect to the
 control plane after a 5-10 second delay.
 
+
+### How does network peering work with Cloud Gateway nodes?
```

Comment:
```suggestion
### How does network peering work with cloud gateway nodes?
```
Just a note, but I did "dedicated cloud gateway" in lowercase, not sure if that's the way to go, but let me know what you think so we can be consistent between our docs.

Reply:
The official blog used Dedicated Cloud Gateway, it's not a good data point but I wonder if "Dedicated Cloud Gateway" should be caps, and "cloud gateway" can be lowercase. https://konghq.com/blog/product-releases/dedicated-cloud-gateways